src/content/experimental/color/color.md

```diff
@@ -1,216 +1,167 @@
 
-![Dark theme example](images/dark-theme-dont-1.png)
+### IBM Base Theme: Light*
 
-![Components with common colors across dark UI backgrounds](images/ui-against-dark-bg.gif)
-Components with common colors across dark UI backgrounds
 
 
 <!--
-<Margin bottom="2rem">
-  <HorizontalRow padded="true">
-    <ColorUIExpandableImage theme="dark" caption="IBM Watson API Demo">
-      <img src="../media/pages/color-ui/color-in-ui-api-dark.jpg" title="IBM Watson API Demo" />
-    </ColorUIExpandableImage>
-    <ColorUIExpandableImage theme="dark" caption="IBM Security Cloud Prototype">
-      <img src="../media/pages/color-ui/color-in-ui-security-dark.jpg" title="IBM Security Cloud Prototype" />
-    </ColorUIExpandableImage>
-    <ColorUIExpandableImage theme="dark" caption="IBM Watson Health Prototype">
-      <img src="../media/pages/color-ui/color-in-ui-health-dark.jpg" title="IBM Watson Health Prototype" />
-    </ColorUIExpandableImage>
-  </HorizontalRow>
-</Margin>
 
--->
+| Color                                                 | Color Name     | Hex Value | Tokens        |
+|-------------------------------------------------------|----------------|-----------|---------------|
+| ![Blue 60](images/swatches/blue-60.png)               | Blue 60        | `0062ff`  | `$brand-01`   |
+| ![Blue 80](images/swatches/blue-80.png)               | Blue 80        | `0530ad`  | `$brand-02`   |
+| ![Blue 60](images/swatches/blue-60.png)               | Blue 60        | `0062ff`  | `$brand-03`   |
+| ![White](images/swatches/white.png)                   | White          | `ffffff`  | `$inverse-01` |
+| ![Gray 10](images/swatches/gray-10.png)               | Gray 10        | `f3f3f3`  | `$ui-01`      |
+| ![White](images/swatches/white.png)                   | White          | `ffffff`  | `$ui-02`      |
+| ![Gray 20](images/swatches/gray-20.png)               | Gray 20        | `dcdcdc`  | `$ui-03`      |
+| ![Gray 50](images/swatches/gray-50.png)               | Gray 50        | `8c8c8c`  | `$ui-04`      |
+| ![Gray 100](images/swatches/gray-100.png)             | Gray 100       | `171717`  | `$ui-05`      |
+| ![Gray 100](images/swatches/gray-100.png)             | Gray 100       | `171717`  | `$text-01`    |
+| ![Gray 70](images/swatches/gray-70.png)               | Gray 70        | `565656`  | `$text-02`    |
+| ![Gray 50](images/swatches/gray-50.png)               | Gray 50        | `8c8c8c`  | `$text-03`    |
+| ![Cool Gray 10](images/swatches/cool-gray-10.png)     | Cool Gray 10   | `f2f4f8`  | `$field-01`   |
+| ![Support Red](images/swatches/support-red.png)       | Support Red    | `D6182D`  | `$support-01` |
+| ![Support Orange](images/swatches/support-orange.png) | Support Orange | `F8792B`  | `$support-02` |
+| ![Support Yellow](images/swatches/support-yellow.png) | Support Yellow | `FBCF47`  | `$support-03` |
+| ![Support Green](images/swatches/support-green.png)   | Support Green  | `30A14C`  | `$support-04` |
```

Comment:
Would be nice to replace all the hex values in this table with the color block.

`<color-block>#30A14C </color-block>`